wis2box-create-config.py

```diff
@@ -358,6 +358,17 @@ def create_wis2box_env(config_dir: str) -> None:
         fh.write('MINIO_NOTIFY_MQTT_TOPIC_WIS2BOX=wis2box/storage\n')
         fh.write('MINIO_NOTIFY_MQTT_QOS_WIS2BOX=1\n')
         fh.write('\n')
+        # downloader settings
```

Comment:
What role do the ENV variables have? Should they not be used here?

Reply:
I write the default values to the wis2box.env to make it easier for the users to edit them